target/linux/ath79/dts/ar7161_netgear_wndr3800.dts

```diff
@@ -141,6 +157,24 @@
 
 &pcie0 {
 	status = "okay";
+
+	ath9k0: wifi@0,11 {
+		compatible = "pci168c,0029";
+		reg = <0x8800 0 0 0 0x10000>;
```

Comment:
does it work with `reg = <0x8800 0 0 0 0x0>;` as well? Some question for the second wifi node.

Reply:
You tested with WNDR3800?   (or just with DIR-825?)

Reply:
I don't think it matters in this case. The bus and pci devices are same.

Reply:
And the 0x10000 is correct. See
https://github.com/openwrt/openwrt/pull/1239#issuecomment-410543236